proto/felixbackend.proto

```diff
@@ -396,3 +410,30 @@ message IPAMPool {
   string cidr = 1;
   bool masquerade = 2;
 }
+
+message ServiceAccountUpdate {
+  ServiceAccountID id = 1;
+  map<string, string> labels = 2;
+}
+
+message ServiceAccountRemove {
+  ServiceAccountID id = 1;
+}
+
+message ServiceAccountID {
+  string namespace = 1;
+  string name = 2;
```

Comment:
Are both namespace and name mandatory? I assume that namespace can be empty (aka 'default').
If u think it will help may be add a comment.

Reply:
We send the default namespace explicitly.